app/.gitignore

```
@@ -45,6 +45,7 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
+app/android/app/.cxx
```

is this the right path? shouldn't it be
```suggestion
android/app/.cxx
```

---

I have made above suggested change by below commit.
https://github.com/acterglobal/a3/pull/2648/commits/ad5dd1c70c1964b49415183356b08fd45fdea6c0